<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<link rel="icon" href="./img/logo.jpeg" type="image/png">
<meta charset="UTF-8">
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />

<meta name="keywords" content="Lantian Yao, CUHK-Shenzhen"> 
<meta name="description" content="Lantian Yao's homepage, CUHK-Shenzhen">
<link rel="stylesheet" href="jemdoc.css" type="text/css">
<title>Lantian Yao's homepage, CUHK-Shenzhen</title>

</head>
<body>
<div id="layout-content" style="margin-top:25px">
<table>
	<tbody>
		<tr>
			<td width="670">
				<div id="toptitle">					
					<h1><font > Lantian Yao (姚岚天) </font></h1>
				</div>
				<p><font > 
                    <h3><font >Email: lantianyao@link.cuhk.edu.cn</font></h3><br>
					Kobilka Institute of Innovative Drug Discovery, (<a href="https://lhs.cuhk.edu.cn/en/page/62">KIDD</a>)<br>
					The Chinese University of Hong Kong, Shenzhen (<a href="https://www.cuhk.edu.cn/en">CUHK-Shenzhen</a>)<br>
                    <h4>[<a href="https://scholar.google.com/citations?user=LQhebNAAAAAJ">Google Sholar</a>]
                        [ORCID: <a href="https://orcid.org/0000-0003-4554-6827">0000-0003-4554-6827</a>]
                        [<a href="https://www.linkedin.com/in/lantian-yao-8b732a285">Linkedin</a>]
                        [<a href="./img/wechat.jpg">Wechat</a>]
                    </h4>
				</font></p>
				<p> 
				    
				    </a>
				</p>
			</td>
			<td>
				  
				<style type="text/css">
					.image1{ 
						border-radius:200px; 
					}
				
					.image2{
						border-radius:20px; 
					}
				</style>
				
				<img class="image1" src="./img/lantian.jpg" border="3" alt="" height="200" align="right" vspace="10" hspace="0"><br>
			</td>
		</tr><tr>
	</tr></tbody>
</table>
<p style="text-align:justify; text-justify:inter-ideograph;">
	<b><span style="font-size:16px;color:red;">I am looking for a faculty or postdoctoral position in the field of bioinformatics and computational biology, with plans to commence in early 2025. If you have any available positions, please feel free to contact me directly at lantianyao@link.cuhk.edu.cn.</span></b>
</p>
<h2><font > Short Bio </font></h2>
<p style="list-style-type:none">
<p style="text-align:justify; text-justify:inter-ideograph;"><font > 
	I am working at the intersection of machine learning and bioinformatics at the <a href="https://lhs.cuhk.edu.cn/en/page/62">Kobilka Institute of Innovative Drug Discovery</a> (Nobel Prize Research Institute) in <a href="https://www.cuhk.edu.cn/en">CUHK-Shenzhen</a>. My primary focus is on the application of artificial intelligence-based methodologies to analyze healthcare data. I am dedicated to advancing the field of drug discovery, with a particular emphasis on peptide drugs, through the development of novel machine learning algorithms.
</font></p>
<p style="text-align:justify; text-justify:inter-ideograph;"><font >
	Furthermore, I have diligently worked to integrate medical and biological data to construct high-quality global biology databases, such as <a href="https://awi.cuhk.edu.cn/~dbAMP/index.php">dbAMP</a> and <a href="https://awi.cuhk.edu.cn/~CircNet/php/index.php">CircNet</a>. My research interests specifically encompass peptide drug discovery, machine and deep learning, biological database design, multi-omics data analysis, and systems biology.
</font></p>
<p style="text-align:justify; text-justify:inter-ideograph;"><font >
To date, I have published <font color="red"><b>23 peer-reviewed papers</b></font> in top journals such as <a style="font-style:italic;"><b>Nucleic Acids Research (IF=16.6), Briefings in Bioinformatics (IF=9.5), Protein Science (IF=8.0), Analytical Chemistry (IF=7.4), Bioinformatics (IF=5.8)</b></a>. These papers have collectively garnered over <font color="red"><b>900 citations</font></b>.
</font></p>
<p>

<h2><font > Recent news </font></h2>
<ul>
<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [November 2024] <a href="https://doi.org/10.1093/nar/gkae1019">A paper has been accepted by Nucleic Acids Research.</a> <br> 
	  </font>
	 </p> </li>	
<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [November 2024] <a href="https://doi.org/10.1016/j.ijbiomac.2024.137668">A paper has been accepted by International Journal of Biological Macromolecules.</a> <br> 
	  </font>
	 </p> </li>	
<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [November 2024] <a href="https://doi.org/10.1093/nar/gkae1005">A paper has been accepted by Nucleic Acids Research.</a> <br> 
	  </font>
	 </p> </li>	


	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [September 2024] <a href="https://doi.org/10.1093/bib/bbae460">A paper has been accepted by Briefings in Bioinformatics.</a> <br> 
	  </font>
	 </p> </li>	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [April 2024] <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.5006">A paper has been accepted by Protein Science. </a><br> 
	  </font>
	 </p> </li>	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
	    [April 2024] <a href="https://academic.oup.com/bib/article/25/3/bbae208/7665137">A paper has been accepted by Briefings in Bioinformatics.</a> <br> 
	  </font>
	 </p> </li>	


</ul>


<h2><font > Publications </font></h2>
<ul>
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://doi.org/10.1093/nar/gkae1019">dbAMP 3.0: updated resource of antimicrobial activity and structural annotation of peptides in the post-pandemic era</a><a href="https://doi.org/10.1093/nar/gkae1019">[Paper]</a> <br> 
         <i>   <b>Lantian Yao</b>, Jiahui Guan, Peilin Xie, Chia-Ru Chung, Zhihao Zhao, Danhong Dong, Yilin Guo, Wenyang Zhang, Junyang Deng, Yuxuan Pang, Yulan Liu, Yunlu Peng, Jorng-Tzong Horng, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	 <i>Nucleic Acids Research, 2024: gkae1019. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>16.6</b></font>)</i> <br>
	  </font>
	 </p> </li>

	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://doi.org/10.1016/j.ijbiomac.2024.137668">DeepKlapred: A deep learning framework for identifying protein lysine lactylation sites via multi-view feature fusion</a><a href="https://doi.org/10.1016/j.ijbiomac.2024.137668">[Paper]</a> <br> 
         <i>   Jiahui Guan, Peilin Xie, Danhong Dong, Qianchen Liu, Zhihao Zhao, Yilin Guo, Yilun Zhang, Tzong-Yi Lee*, <b>Lantian Yao*</b>, Ying-Chih Chiang* (*Corresponding authors)</i><br> 
	 <i>International Journal of Biological Macromolecules, 2024. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>7.7</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://doi.org/10.1093/nar/gkae1005">dbPTM 2025 update: comprehensive integration of PTMs and proteomic data for advanced insights into cancer research</a><a href="https://doi.org/10.1093/nar/gkae1005">[Paper]</a> <br> 
         <i>   Chia-Ru Chung, Yun Tang, Yen-Peng Chiu, Shangfu Li, Wen-Kai Hsieh, <b>Lantian Yao</b>, Ying-Chih Chiang, Yuxuan Pang, Guan-Ting Chen, Kai-Chen Chou, You Sheng Paik, Phuong Lam Tran, Cheng-Pei Lin, Yu-Min Kao, Yi-Jie Chen, Wen-Chi Chang, Justin Bo-Kai Hsu, Jorng-Tzong Horng, Tzong-Yi Lee</i><br> 
	 <i>Nucleic Acids Research, 2024: gkae1005. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>16.6</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://doi.org/10.1093/bib/bbae460">ACP-CapsPred: an explainable computational framework for identification and functional prediction of anticancer peptides based on capsule network</a><a href="https://doi.org/10.1093/bib/bbae460">[Paper]</a> <br> 
         <i>   <b>Lantian Yao</b>, Peilin Xie, Jiahui Guan, Chia-Ru Chung, Wenyang Zhang, Junyang Deng, Yixian Huang, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	 <i>Briefings in Bioinformatics, 2024, 25(5): bbae460. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>9.5</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.4c00546">CapsEnhancer: An Effective Computational Framework for Identifying Enhancers Based on Chaos Game Representation and Capsule Network</a><a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.4c00546">[Paper]</a> <br> 
         <i>   <b>Lantian Yao</b>, Peilin Xie, Jiahui Guan, Chia-Ru Chung, Yixian Huang, Yuxuan Pang, Huacong Wu, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	 <i>	Journal of Chemical Information and Modeling 64.14 (2024): 5725-5736. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.6</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.5006">AMPActiPred: A three-stage framework for predicting antibacterial peptides and activity levels with deep forest</a><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.5006">[Paper]</a> <br> 
         <i>   <b>Lantian Yao</b>, Jiahui Guan, Peilin Xie, Chia-Ru Chung, Junyang Deng, Yixian Huang, Ying-Chih Chiang, Tzong-Yi Lee </i><br> 
	 <i>	Protein Science, 2024, 33(6): e5006. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>8.0</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://academic.oup.com/bib/article/25/3/bbae208/7665137">A two-stage computational framework for identifying antiviral peptides and their functional types based on contrastive learning and multi-feature fusion strategy</a> <a href="https://academic.oup.com/bib/article/25/3/bbae208/7665137">[Paper]</a> <br> 
         <i>Jiahui Guan†, <b>Lantian Yao† (†Co-first Author)</b>, Peilin Xie, Chia-Ru Chung, Yixian Huang, Ying-Chih Chiang, Tzong-Yi Lee </i><br> 
	 <i>Briefings in Bioinformatics, 2024, 25(3): bbae208. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>9.5</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li> <p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.3c04196">Identifying Antitubercular Peptides via Deep Forest Architecture with Effective Feature Representation</a> <a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.3c04196">[Paper]</a> <br> 
         <i><b>Lantian Yao</b>, Jiahui Guan, Wenshuo Li, Chia-Ru Chung, Junyang Deng, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	 <i>Analytical Chemistry, 2024, 96(4): 1538-1546. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>7.4</b></font>)</i> <br>
	  </font>
	 </p> </li>
	
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://www.sciencedirect.com/science/article/pii/S2589004224005546">KinPred-RNA-Kinase Activity Inference and Cancer Type Classification using Machine Learning on RNA-seq Data</a> <a href="https://www.sciencedirect.com/science/article/pii/S2589004224005546">[Paper]</a> <br> 
         <i> Yuntian Zhang†, <b>Lantian Yao† (†Co-first Author)</b>, Chia-Ru Chung, Yixian Huang, Shangfu Li, Wenyang Zhang, Yuxuan Pang, Tzong-Yi Lee</i><br> 
	 <i>iScience, 2024. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.8</b></font>)</i> <br>
	  </font>
	 </p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
            <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.4758">DeepAFP: An effective computational framework for identifying antifungal peptides based on deep learning</a> <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.4758">[Paper]</a> <br> 
         <i> <b>Lantian Yao</b>, Yuntian Zhang, Wenshuo Li, Chia-Ru Chung, Jiahui Guan, Wenyang Zhang, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	 <i>Protein Science, 2023, 32(10): e4758 (SCI, JCR:Q1, Impact Factor: <font color="red"><b>8.0</b></font>)</i> <br>
	  </font>
	 </p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/nar/article/50/D1/D460/6445964">dbAMP 2.0: updated resource for antimicrobial peptides with an enhanced scanning method for genomic and proteomic data</a> <a href="https://academic.oup.com/nar/article/50/D1/D460/6445964">[Paper]</a> <br> 
	 <i> Jhih-Hua Jhong†, <b>Lantian Yao† (†Co-first Author)</b>, Yuxuan Pang, Zhongyan Li, Chia-Ru Chung, Rulan Wang, Shangfu Li, Wenshuo Li, Mengqi Luo, Renfei Ma, Yuqi Huang, Xiaoning Zhu, Jiahong Zhang, Hexiang Feng, Qifan Cheng, Chunxuan Wang, Kun Xi, Li-Ching Wu, Tzu-Hao Chang, Jorng-Tzong Horng, Lizhe Zhu, Ying-Chih Chiang, Zhuo Wang, Tzong-Yi Lee</i><br> 
	<i>Nucleic acids research, 2022, 50(D1): D460-D470. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>14.9</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/nar/article/50/D1/D93/6444342">CircNet 2.0: an updated database for exploring circular RNA regulatory networks in cancers</a> <a href="https://academic.oup.com/nar/article/50/D1/D93/6444342">[Paper]</a> <br> 
	 <i>Yigang Chen†, <b>Lantian Yao† (†Co-first Author)</b>, Yun Tang, Jhih-Hua Jhong, Jingting Wan, Jingyue Chang, Shidong Cui, Yijun Luo, Xiaoxuan Cai, Wenshuo Li, Qi Chen, Hsi-Yuan Huang, Zhuo Wang, Weiming Chen, Tzu-Hao Chang, Fengxiang Wei, Tzong-Yi Lee, Hsien-Da Huang</i><br> 
	<i>Nucleic acids research, 2022, 50(D1): D93-D101. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>14.9</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://www.mdpi.com/1422-0067/24/5/4328">Accelerating the discovery of anticancer peptides through deep forest architecture with deep graphical representation</a> <a href="https://www.mdpi.com/1422-0067/24/5/4328">[Paper]</a> <br> 
	 <i><b>Lantian Yao</b>, Wenshuo Li, Yuntian Zhang, Junyang Deng, Yuxuan Pang, Yixian Huang, Chia-Ru Chung, Jinhan Yu, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	<i>International Journal of Molecular Sciences, 2023, 24(5): 4328. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.6</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/bioinformatics/article/38/24/5368/6795008">Integrating transformer and imbalanced multi-label learning to identify antimicrobial peptides and their functional activities</a> <a href="https://academic.oup.com/bioinformatics/article/38/24/5368/6795008">[Paper]</a> <br> 
	 <i>Yuxuan Pang†, <b>Lantian Yao† (†Co-first Author)</b>, Jingyi Xu, Zhuo Wang, Tzong-Yi Lee</i><br> 
	<i>Bioinformatics, 2022, 38(24): 5368-5374. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.8</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://www.mdpi.com/2076-3417/13/12/6965">ABPCaps: A Novel Capsule Network-Based Method for the Prediction of Antibacterial Peptides</a> <a href="https://www.mdpi.com/2076-3417/13/12/6965">[Paper]</a> <br> 
	 <i><b>Lantian Yao</b>, Yuxuan Pang, Jingting Wan, Chia-Ru Chung, Jinhan Yu, Jiahui Guan, Clement Leung, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	<i>Applied Sciences, 2023, 13(12): 6965. (SCI, JCR:Q1, Impact Factor: 2.7)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.3c01602">Predicting Anti-inflammatory Peptides by Ensemble Machine Learning and Deep Learning</a> <a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.3c01602">[Paper]</a> <br> 
	 <i>Jiahui Guan†, <b>Lantian Yao† (†Co-first Author)</b>, Chia-Ru Chung, Peilin Xie, Yilun Zhang, Junyang Deng, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	<i>Journal of Chemical Information and Modeling, 2023, 63(24): 7886-7898. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.6</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/gpb/article/21/1/228/7274153">KinasePhos 3.0: redesign and expansion of the prediction on kinase-specific phosphorylation sites</a> <a href="https://academic.oup.com/gpb/article/21/1/228/7274153">[Paper]</a> <br> 
	 <i>Renfei Ma, Shangfu Li, Wenshuo Li, <b>Lantian Yao</b>, Hsien-Da Huang, Tzong-Yi Lee</i><br> 
	<i>Genomics, Proteomics and Bioinformatics, 2023, 21(1): 228-241. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>9.5</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://www.mdpi.com/1422-0067/24/12/10348">StackTHPred: Identifying Tumor-Homing Peptides through GBDT-Based Feature Selection with Stacking Ensemble Architecture</a> <a href="https://www.mdpi.com/1422-0067/24/12/10348">[Paper]</a> <br> 
	 <i>Jiahui Guan†, <b>Lantian Yao† (†Co-first Author)</b>, Chia-Ru Chung, Ying-Chih Chiang, Tzong-Yi Lee</i><br> 
	<i>International Journal of Molecular Sciences, 2023, 24(12): 10348. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.6</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/bib/article/22/6/bbab263/6323205">AVPIden: a new scheme for identification and functional prediction of antiviral peptides based on machine learning approaches</a> <a href="https://academic.oup.com/bib/article/22/6/bbab263/6323205">[Paper]</a> <br> 
	 <i>Yuxuan Pang, <b>Lantian Yao</b>, Jhih-Hua Jhong, Zhuo Wang, Tzong-Yi Lee</i><br> 
	<i>Briefings in bioinformatics, 2021, 22(6): bbab263 (SCI, JCR:Q1, Impact Factor: <font color="red"><b>9.5</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/bib/article/24/1/bbac497/6847797">Extraction of microRNA–target interaction sentences from biomedical literature by deep learning approach</a> <a href="https://academic.oup.com/bib/article/24/1/bbac497/6847797">[Paper]</a> <br> 
	 <i>Mengqi Luo, Shangfu Li, Yuxuan Pang, <b>Lantian Yao</b>, Renfei Ma, Hsi-Yuan Huang, Hsien-Da Huang, Tzong-Yi Lee</i><br> 
	<i>Briefings in Bioinformatics, 2023, 24(1): bbac497 (SCI, JCR:Q1, Impact Factor: <font color="red"><b>9.5</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/nar/article/50/D1/D471/6426061">dbPTM in 2022: an updated database for exploring regulatory networks and functional associations of protein post-translational modifications</a> <a href="https://academic.oup.com/nar/article/50/D1/D471/6426061">[Paper]</a> <br> 
	 <i>Zhongyan Li, Shangfu Li, Mengqi Luo, Jhih-Hua Jhong, Wenshuo Li, <b>Lantian Yao</b>, Yuxuan Pang, Zhuo Wang, Rulan Wang, Renfei Ma, Jinhan Yu, Yuqi Huang, Xiaoning Zhu, Qifan Cheng, Hexiang Feng, Jiahong Zhang, Chunxuan Wang, Justin Bo-Kai Hsu, Wen-Chi Chang, Feng-Xiang Wei, Hsien-Da Huang, Tzong-Yi Lee</i><br> 
	<i>Nucleic acids research, 2022, 50(D1): D471-D479. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>14.9</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://academic.oup.com/nar/article/50/D1/D222/6446528">miRTarBase update 2022: an informative resource for experimentally validated miRNA-target interactions</a> <a href="https://academic.oup.com/nar/article/50/D1/D222/6446528">[Paper]</a> <br> 
	 <i>Hsi-Yuan Huang, Yang-Chi-Dung Lin, Shidong Cui, Yixian Huang, Yun Tang, Jiatong Xu, Jiayang Bao, Yulin Li, Jia Wen, Huali Zuo, Weijuan Wang, Jing Li, Jie Ni, Yini Ruan, Liping Li, Yidan Chen, Yueyang Xie, Zihao Zhu, Xiaoxuan Cai, Xinyi Chen, <b>Lantian Yao</b>, Yigang Chen, Yijun Luo, Shupeng LuXu, Mengqi Luo, Chih-Min Chiu, Kun Ma, Lizhe Zhu, Gui-Juan Cheng, Chen Bai, Ying-Chih Chiang, Liping Wang, Fengxiang Wei, Tzong-Yi Lee, Hsien-Da Huang</i><br> 
	<i>Nucleic acids research, 2022, 50(D1): D222-D230. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>14.9</b></font>)</i> <br>
	</font>
	</p>
	</li>
	<li><p style="margin-left: 0px; line-height: 150%; margin-top: 10px; margin-bottom: 10px;"><font size="3"><meta charset="utf-8">
		<a href="https://www.mdpi.com/1422-0067/24/18/14061">A robust drug-target interaction prediction framework with capsule network and transfer learning</a> <a href="https://www.mdpi.com/1422-0067/24/18/14061">[Paper]</a> <br> 
	 <i>Yixian Huang, Hsi-Yuan Huang, Yigang Chen, Yang-Chi-Dung Lin, <b>Lantian Yao</b>, Tianxiu Lin, Junlin Leng, Yuan Chang, Yuntian Zhang, Zihao Zhu, Kun Ma, Yeong-Nan Cheng, Tzong-Yi Lee, Hsien-Da Huang</i><br> 
	<i>International Journal of Molecular Sciences, 2023, 24(18): 14061. (SCI, JCR:Q1, Impact Factor: <font color="red"><b>5.6</b></font>)</i> <br>
	</font>
	</p>
	</li>
</ul>

<h2><font > Reviewer </font></h2>
<ul style="list-style-type:none">
   <p style="margin-left: 0px; line-height: 150%; margin-top: 8px; margin-bottom: 8px;"><font>
		<i>Microbiome</i> <br>
	        <i>Protein Science</i> <br>
		<i>Scientific Report</i><br>
		<i>Gene</i><br>
	        <i>BMC Bioinformatics</i><br>
		<i>The Journal of Supercomputing</i><br>
		<i>Mathematical Biosciences and Engineering</i><br>
		<i>Bioinformatics Advances</i><br>
   </font></p>
</ul>
</div>
<script type="text/javascript" id="clustrmaps" src="//clustrmaps.com/map_v2.js?d=tcfNfbLvhLKOe5MczD7jyQmovcroK1X0gqtHPqc69tM&cl=ffffff&w=a"></script>
</body>
